src/aiida/engine/processes/control.py

```diff
@@ -201,7 +202,7 @@ def kill_processes(
         return
 
     controller = get_manager().get_process_controller()
-    action = functools.partial(controller.kill_process, msg_text=msg_text)
+    action = functools.partial(controller.kill_process, msg_text=msg_text, force_kill=force_kill)
```

Comment:
In my opinion, this is one of the changes need to be tested. The CLI tests are calling this and should not test against using daemon, which cross too many boundaries. 

In essence, if I understand correctly to make `verdi process kill --force` work, it need:
- `--force` can propagate to the `kill_processes` to make the `force_kill` set to `True`
- kill_processes function should send the message through RMQ with message resolved with `force` as an attriute of the function. 
- Then it is plumpy part (override by `aiida-core/engine/processes/process.py::Process::kill`, this I said the evil of inheritance) need to make sure it calls kill properly, so it is out the scope of the tests covered in aiida-core.

Reply:
This should be now tested.